docs/CONFIG-VARS.md

```diff
@@ -61,7 +61,7 @@ You can use `default_public_access_cidrs` to set a default range for all created
 | gke_control_plane_subnet_cidr |  Address space for the hosted master subnet | string | "10.2.0.0/28" | When providing your own subnets (by setting `subnet_names` make sure your subnets do not overlap this range  |
 | misc_subnet_cidr | Address space for the the auxiliary resources (Jump VM and optionally NFS VM) subnet | string | "192.168.2.0/24" | This variable is ignored when `subnet_names` is set (aka bring your own subnet) |
 | filestore_subnet_cidr | Address space for Google Filestore subnet | string | "192.168.3.0/29" | Needs to be at least a /29 range. Only used when `storage_type="ha"` |
-| database_subnet_cidr | Address space for Google Cloud SQL Postgre subnet | string | "192.168.4.0/24" | Only used when `create_postgres=true` |
+| database_subnet_cidr | Address space for Google Cloud SQL Postgres subnet | string | "192.168.4.0/24" | Only used with external postgres |
```

Comment:
This code is infrastructure so why are we trying to tell them this is an external postgres? This is a SAS related term and relevant to SAS Viya. The original comment tells the user this code is creating a postgres server if the flag `create_postgres=true` I do not believe this should be changed. Please clarify the reason for the update.